solana/program_v2/packages/client/core/src/GatekeeperService.ts

```diff
@@ -108,6 +108,11 @@ export class GatekeeperService extends AbstractService {
   issue(
     passAccount: PublicKey,
     subject: PublicKey,
+    splToken?: PublicKey,
```

Comment:
Should these be optional?

Reply:
In the case of native SOL it would be ideal... though on the program side it doesn't work (with anchor at least)

There is an option to manually parse "remaining_accounts" (which are kinda unspecified additional accounts) - feels like a lot of work

Reply:
To be specific, doing Option<Account> on the context fails

Reply:
What's the difference between `mint` and `splToken`?

Reply:
Generally I would be ok to have an optional argument around `splToken`. Maybe we should consider `token-2022` in this interface as well. It's out already and I think Gateway V2 should support it. @JulianSpring could you have a look about how much complexity that would be? Maybe let's create a dedicated ticket for it.

Reply:
👍